### PR TITLE
Add BERTopic intergration information to docs

### PR DESCRIPTION
**Pull request by davanstrien**

### CODE REVIEW

**Comment by davanstrien** [comment]
cc @MaartenGr I suggest we wait with merging this until you are ready to make a release for BERTopic with this integration. 

@osanseviero I think I've added everything for this part. At the moment, I've indicated the widgets/ API doesn't work. I would suggest updating that once the PR in https://github.com/huggingface/api-inference-community/pull/276 is merged (which would also rely on a new release of BERTopic with the integration.

File: js/src/lib/interfaces/Libraries.ts [hunk 1]
```diff
@@ -105,6 +106,11 @@ const asteroid = (model: ModelData) =>
 
 model = BaseModel.from_pretrained("${model.id}")`;
 
+const bertopic = (model: ModelData) =>
```
Comment (osanseviero) [inline js/src/lib/interfaces/Libraries.ts:109]: You then need to add this to `MODEL_LIBRARIES_UI_ELEMENTS` below

**Review by osanseviero — COMMENTED** [review]
Thanks for this!

**Review by osanseviero — COMMENTED** [review]
Thanks for this!

**Comment by davanstrien** [comment]
@osanseviero think this is ready to be merged :)

**Review by osanseviero — APPROVED** [review]
Thanks!

**Pull request merged**